doc/source/whatsnew/v1.1.0.rst

```diff
@@ -380,6 +380,35 @@ Assignment to multiple columns of a :class:`DataFrame` when some of the columns
    df[['a', 'c']] = 1
    df
 
```

Comment:
Can you move the note to the bug fix section? I don’t think this is API breaking

Reply:
This was requested by @jreback - thoughts?

Reply:
I reverted the whatsnew back to a bug fix instead of API breaking, but can go back if there are thoughts otherwise.